bcipy/tasks/rsvp/icon_to_icon.py

```diff
@@ -453,6 +456,59 @@ def _init_icon_to_icon_display_task(parameters, win, daq, static_clock,
         is_word=is_word)
 
 
+class RandomLm():
```

Comment:
Since this is LM specific, I'd vote to move it to either helpers under lm_related or within the language_model module (and it could be the default.)

Reply:
I didn't want to pollute the core codebase with task-specific code, but I agree that this can be more generally useful. I factored it out into the language_model module. At some point we may want to parameterize it with the type of distribution to use. For this task it is using a uniform distribution, but there were also some interesting experiments that can conducted with a normal distribution.